itests/src/test/java/fi/metatavu/edelphi/test/ui/base/panel/QueryTestsBase.java

```diff
@@ -0,0 +1,201 @@
+package fi.metatavu.edelphi.test.ui.base.panel;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import fi.metatavu.edelphi.test.mock.FolderMocker;
+import fi.metatavu.edelphi.test.mock.PanelMocker;
+import fi.metatavu.edelphi.test.mock.QueryMocker;
+import fi.metatavu.edelphi.test.ui.base.AbstractUITest;
+
+public class QueryTestsBase extends AbstractUITest {
+  
+  private FolderMocker folderMocker;
+  private QueryMocker queryMocker;
+  private PanelMocker panelMocker;
+  
+  @Before
+  public void before() {
+    folderMocker = new FolderMocker();
+    queryMocker = new QueryMocker();
+    panelMocker = new PanelMocker();
+  }
+  
+  @After
+  public void after() {
+    queryMocker.cleanup();
+    folderMocker.cleanup();
+    panelMocker.cleanup();
+  }
+  
+  @Test
+  public void testPageWalk() {
+    login(ADMIN_EMAIL);
+    createTestPanel();
+    
+    createTestQuery("test", "basic-text-page", "basic-expertise-page", "basic-scale1d-page", "basic-scale2d-page", 
+        "basic-timeserie-page", "basic-timeline-page", "basic-grouping-page", "basic-multiselect-page", "basic-order-page", 
+        "basic-form-page", "basic-background-form-page", "basic-collage2d-page");
+    
+    navigate("/test/test-query");
+    
+    // Text
+    
+    waitAndAssertText(".queryPageTitle", "Text page");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal ".queryPageTitle" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)